api/app/signals/apps/questionnaires/fieldtypes/selected_object.py

```diff
@@ -0,0 +1,85 @@
+# SPDX-License-Identifier: MPL-2.0
+# Copyright (C) 2022 Gemeente Amsterdam
+from signals.apps.questionnaires.fieldtypes.base import FieldType
+
+
+class SelectedObject(FieldType):
+    verbose_name = 'Selected object'
+    submission_schema = {
+        'type': 'object',
+        'properties': {
```

Comment:
heeft als property ook nog een category_url

Reply:
Dit is nu het geval in de extra properties, ik weet niet of we dit nog willen behouden. Ik zal hem er niet verplicht bij zetten voor nu.